HandleNHCreateOrUpdateInstallationCallOrchestrator/handler.ts

```diff
@@ -53,17 +62,36 @@ export const getHandler = ({
         installationId
       });
 
-      logger.info(
-        `INSTALLATION_ID:${installationId}|IS_TEST_USER:${isUserATestUser.value}`
-      );
+      if (isUserATestUser) {
+        logger.info(`TEST_USER:${installationId}`);
 
-      yield* createOrUpdateActivity(context, {
-        installationId,
-        notificationHubConfig,
-        platform,
-        pushChannel,
-        tags
-      });
+        const notificationHubConfigPartition = notificationHubConfigPartitionChooser(
+          installationId
+        );
+
+        yield* createOrUpdateActivity(context, {
+          installationId,
+          notificationHubConfig: notificationHubConfigPartition,
+          platform,
+          pushChannel,
+          tags
+        });
+
+        // Always delete installation from legacy Notification Hub
+        yield* deleteInstallationActivity(context, {
+          installationId,
+          notificationHubConfig: legacyNotificationHubConfig
+        });
```

Comment:
New `CreateOrUpdate` management, if user is enabled for test:
- call `createOrUpdate` into new NH
- call `deleteInstallation` into legacy NH

Reply:
TODO: add try/catch in case creation will fail and a telemetry to be notified about it